platform-includes/performance/default-sampling-context-platform/unreal.mdx

```diff
@@ -0,0 +1 @@
+For Unreal Engine SDK, it includes a [Transaction Context](https://github.com/getsentry/sentry-unreal/blob/main/plugin-dev/Source/Sentry/Public/SentryTransactionContext.h) and a custom sampling context (`FString` to `FString` map).
```

Comment:
"For Unreal Engine SDK" seems off. What are we trying to say? (I guess depends on where this file is included?)

Also note that we're linking to a file under a branch name so this link will 404 if we move things.
Consider pinning this to a commit sha:

```suggestion
The SDK for Unreal Engine includes a [Transaction Context](https://github.com/getsentry/sentry-unreal/blob/6e4ee91fd538cede13e95635dd1e38c3ad0ab0ba/plugin-dev/Source/Sentry/Public/SentryTransactionContext.h) and a custom sampling context (`FString` to `FString` map).
```

Reply:
I've rephrased it for better clarity:
```
`SamplingContext` includes a `Transaction Context` and a custom sampling context (`FString` to `FString` map).
```

Good point about linking a file, thanks!